cmake/ccf_snp_unsafe-config.cmake.in

```
@@ -0,0 +1,9 @@
+@PACKAGE_INIT@
```

As discussed, let's only go for a the unsafe variant for SGX (we should throw an error if UNSAFE_VERSION is passed with a compile target != `sgx`).

---

#4441